resources/manager/data.go

```diff
@@ -19,39 +19,42 @@ package manager
 
 import (
 	"context"
+	"fmt"
 	"sync"
 	"time"
 
-	"github.com/elastic/cloudbeat/resources/fetching"
-
 	"github.com/elastic/beats/v7/libbeat/logp"
+	"github.com/elastic/cloudbeat/resources/fetching"
 )
 
 // Data maintains a cache that is updated by Fetcher implementations registered
 // against it. It sends the cache to an output channel at the defined interval.
 type Data struct {
-	interval  time.Duration
-	output    chan ResourceMap
-	cycleData ResourceMap
-	fetchers  FetchersRegistry
-	wg        *sync.WaitGroup
+	timeout  time.Duration
```

Comment:
Consider renaming the struct to `Manager`

Reply:
`Manger` is widely used in the agent, maybe something more specific?